cves/CVE-2016-8740.yml

```diff
@@ -49,6 +51,10 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
+  A specific module, the modhttp2 module did not offer data length
+  restriction for the request headers. This means that malicious sources could potentially
+  send through large amounts of information, and block out threads, stopping others
+  from being able to use it.
```

Comment:
Is a denial of service attack could be accomplished? May want to state that explicitly if that is the case.

Reply:
good idea, ill be sure to include it